### ISSUE TITLE
EXIF info not recognized in HEIC image

### ISSUE THREAD

**Issue by mshgh** [MikeKovarik/exifr#35]
In this photo [Sample.heic](https://github.com/mshgh/stackblitz-ha2-demos/blob/master/Sample.heic?raw=true), taken by Samsung Galaxy A40, there was not found any EXIF info using this library.
I know there is EXIF info in it as Windows 10 and imagemagick can see it.

**Comment by MikeKovarik** [comment]
Hi, so it should be fixed right now. I just released 5.0.4!

**Comment by mshgh** [comment]
Thank you for the quick response. Unfortunately upgrade to the new version didn't help for me. I have prepared simple test app and it doesn't work either. Here is my code. (node version *v12.18.3*) Any idea what I am doing wrong?

*index.mjs*
```javascript
import {promises as fs} from 'fs'
import exifr from 'exifr'

async function main() {
  const file = await fs.readFile('/tmp/issue-exifr-35.heic')
  const output = await exifr.parse(file, true)
  console.log('exifr:', output)
}
```
*package.json*
```javascript
{
  "name": "test",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "exifr": "^5.0.4"
  }
}
```
To be sure I use the same file I re-downloaded the image
```
curl https://raw.githubusercontent.com/MikeKovarik/exifr/master/test/fixtures/issue-exifr-35.heic --output /tmp/issue-exifr-35.heic
```

**Comment by MikeKovarik** [comment]
Aaaah, you're not doing anything wrong. It's my bad. I accidentally released 5.0.4. with code from 5.0.3.

So i released 5.0.5. and tested it with your code and it works now.

**Comment by mshgh** [comment]
Works fine for me now. Thanks!

**Comment by mshgh** [comment]
Heads-up, I got exception when using
```
exifr.parse('/tmp/issue-exifr-35.heic', true).then(console.log).catch(console.log)

RangeError: Offset is outside the bounds of the DataView
    at DataView.getUint32 (<anonymous>)
    at Object.getUint32 (/root/test/node_modules/exifr/dist/full.umd.js:1:3533)
    at Oe.parseBoxHead (/root/test/node_modules/exifr/dist/full.umd.js:1:29964)
    at Oe.parse (/root/test/node_modules/exifr/dist/full.umd.js:1:30474)
    at q.parse (/root/test/node_modules/exifr/dist/full.umd.js:1:11345)
    at Object.Q [as parse] (/root/test/node_modules/exifr/dist/full.umd.js:1:12078)
```
Not sure if this is connected to the recent change, or not.

**Comment by MikeKovarik** [comment]
Hello @mshgh . I'm sorry, I didn't notice the last comment in closed issue. I just fixed that and released 5.0.6
